_sass/mei.scss

```diff
@@ -393,11 +393,21 @@ ul.nav li.nav-item:first-child {
         content: "Goal: "
     }
 
+    .tutorialGoalES:before {
+        font-weight: bold;
+        content: "Objetivo: "
+    }
+
     .task:before {
         font-weight: bold;
         content: "Task: "
     }
 
+    .task:beforeES {
```

Comment:
```suggestion
    .task-ES:before {
```

Reply:
`task:beforeES` will not work, right?

Suggestion to use hyphen as in file names. Would need to be adjusted in all classes in html files.